kitsune/sumo/utils.py

```diff
@@ -13,9 +13,13 @@
 from ratelimit.utils import is_ratelimited as rl_is_ratelimited
 from timeout_decorator import timeout
 
+from kitsune.lib.tlds import VALID_TLDS
 from kitsune.journal.models import Record
 from kitsune.sumo import paginator
 
+POTENTIAL_LINK_REGEX = re.compile(r'[^\s/]+\.([^\s/.]{2,})')
```

Comment:
This isn't a setting though: its format is inherently tied to the behaviour of `has_blocked_link` - if the capturing groups change, for instance, the function will behave differently.

Reply:
but how is the function related to the regex? My proposal is to mainly group all regexes related to spam together. Marking this as nit

Reply:
Because the function's behaviour relies on the regex being in this exact form, with this exact capturing group. If the capturing groups were changed, the function would cease to work. In my mind if something's in settings.py then it should be configurable, and the configurable aspect of this function is the allowlist, not the regexp.